homeassistant/components/emulated_hue/upnp.py

```diff
@@ -119,7 +107,7 @@ def run(self):
             socket.inet_aton(self.host_ip_addr))
 
         if self.upnp_bind_multicast:
-            ssdp_socket.bind(("239.255.255.250", 1900))
+            ssdp_socket.bind(("", 1900))
```

Comment:
Reasonably sure.
All the searching I did, showed that a sendto needs to use the multicast ip address.
receiving of socket messages has to be via an ip address associated with a local adapter, thus you can use a specific ip locally bound, or use the any, if the device has multiple ip addresses.

Since the multicast address cannot be bound to an adapter, its not suitable as an address for receiving.  Perhaps the linux code is smarter, and ignores multicast addresses when binding to a socket for receiving.
eg
https://www.electricmonk.nl/log/2016/07/05/exploring-upnp-with-python/
or a dotnet version
https://stackoverflow.com/questions/12794761/upnp-multicast-missing-answers-from-m-search-discovery